src/passes/MergeBlocks.cpp

```diff
@@ -223,10 +224,17 @@ struct MergeBlocks : public WalkerPass<PostWalker<MergeBlocks>> {
     if (auto* block = child->dynCast<Block>()) {
       if (!block->name.is() && block->list.size() >= 2) {
         child = block->list.back();
+        // we modified child )which is *&), which modifies curr, which might change its type
```

Comment:
Typo: flipped `(` before "which"

Actually what does `*&` mean in  `(which is *&)`?
Oh, "a pointer reference." Might be better phrased "which is *a* *&", or "which is a reference"